packages/forma-36-react-components/src/components/Skeleton/SkeletonRow/SkeletonTableCell/SkeletonTableCell.tsx

```diff
@@ -0,0 +1,35 @@
+import React, { Component } from 'react';
+
+import TableCell from '../../../Table/TableCell';
+import SkeletonContainer from '../../SkeletonContainer';
+import SkeletonBodyText from '../../SkeletonBodyText';
+
+export type SkeletonTableCellProps = {
+  clipId?: string;
+} & typeof defaultProps;
+
+const defaultProps = {
+  clipId: 'skeleton-table-cell',
+};
+
+export class SkeletonTableCell extends Component<SkeletonTableCellProps> {
+  static defaultProps = defaultProps;
+
+  render() {
+    const { clipId } = this.props;
+
+    return (
+      <TableCell>
+        <SkeletonContainer
+          svgHeight={42}
```

Comment:
@m10l if I remove the svgHeight this is what happens

![Screenshot 2020-03-19 at 17 50 24](https://user-images.githubusercontent.com/6597467/77092793-67170300-6a0a-11ea-8f60-1182180c3675.png)

The reason for this is that the `<rect>` in the svg gets the default height which is 100%

What do you think we should do?